talend-component-maven-plugin/src/it/web/test/tck-action-user-api-test/tck_action_user_api_test.json

```diff
@@ -0,0 +1,155 @@
+    {
+      "entity": {
+        "type": "Project",
+        "description": "To run the test you need to run a component server.  \nTesting documentation can be found [here](https://github.com/Talend/component-runtime/tree/master/talend-component-maven-plugin/src/it/web)  \nApi doc is [here for action execute](https://talend.github.io/component-runtime/main/latest/rest-openapi.html#/Action/execute)  \nAction documentation is [here for user action](https://talend.github.io/component-runtime/main/latest/services-actions.html#_user)",
+        "id": "6c28fa92-59a7-47a6-818b-0e5de4314d58",
+        "name": "tck-action-user-api-test"
+      },
+      "children": [
+        {
+          "entity": {
+            "type": "Request",
+            "method": {
+              "requestBody": true,
+              "link": "http://tools.ietf.org/html/rfc7231#section-4.3.3",
+              "name": "POST"
+            },
+            "body": {
+              "formBody": {
+                "overrideContentType": true,
+                "encoding": "application/x-www-form-urlencoded",
+                "items": []
+              },
+              "bodyType": "Text",
+              "textBody": "{}"
+            },
+            "uri": {
+              "query": {
+                "delimiter": "&",
+                "items": [
+                  {
+                    "enabled": true,
+                    "name": "family",
+                    "value": "the_family"
+                  },
+                  {
+                    "enabled": true,
+                    "name": "type",
+                    "value": "user"
+                  },
+                  {
+                    "enabled": true,
+                    "name": "action",
+                    "value": "action_USER"
+                  }
+                ]
+              },
+              "scheme": {
+                "name": "http",
+                "version": "V11"
+              },
+              "host": "${\"server-ip\"}:${\"server-port\"}",
+              "path": "/api/v1/action/execute"
+            },
+            "id": "2da7e306-dc62-4af1-8a9d-caca22c40f87",
+            "name": "1. action.execute - user action check",
+            "headers": [
+              {
+                "enabled": true,
+                "name": "Content-Type",
+                "value": "application/json"
+              }
+            ],
+            "assertions": [
+              {
+                "comparison": "Equals",
+                "subject": "ResponseStatus",
+                "path": "code",
+                "value": "200"
+              },
+              {
+                "comparison": "Equals",
+                "subject": "ResponseJsonBody",
+                "path": "$.username",
+                "value": "talend"
+              },
+              {
+                "comparison": "Equals",
+                "subject": "ResponseJsonBody",
+                "path": "$.password",
+                "value": "123456"
```

Comment:
```suggestion
                "value": "\"123456\""
```
To avoid assertion warning 
![image](https://github.com/Talend/component-runtime/assets/1255625/662622d4-596a-4741-b785-9bf593c6a55c)
![image](https://github.com/Talend/component-runtime/assets/1255625/980a8f01-d2d5-40ed-81d1-6b3e12be2c66)